src/navbar/navbar.styles.tsx

```diff
@@ -33,7 +33,8 @@ export const Wrapper = styled.div<StyleProps>`
     left: 0;
     right: 0;
     width: 100%;
-    box-shadow: 0 1px 3px rgba(0, 0, 0, 0.1);
+    border-bottom: ${Border["width-010"]} ${Border["solid"]} ${Colour["border"]};
```

Comment:
will need to account for the extra 1px height introduced by the border

the Nav `height` should be `calc(${xxx}rem - 1px)` to compensate